src/message-backup/BackupClient.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Where message backups should be stored
+ */
+export enum BackupProvider {
```

Comment:
this is nitpick, but I'd expect `BackupProvider` to be an interface or something more substantial. This seems to be a provider id or type at this point. Do you expect this to evolve into something bigger down the line?